dhctl/pkg/operations/mirror/pull.go

```diff
@@ -131,10 +148,26 @@ func PullImageSet(
 func PullModules(mirrorCtx *Context, layouts *ImageLayouts) error {
 	log.InfoLn("Beginning to pull Deckhouse modules")
 	for moduleName, moduleData := range layouts.Modules {
-		if err := PullImageSet(mirrorCtx.RegistryAuth, moduleData.ModuleLayout, moduleData.ModuleImages, mirrorCtx.Insecure, mirrorCtx.SkipTLSVerification, false); err != nil {
+		if err := PullImageSet(
+			mirrorCtx.RegistryAuth,
```

Comment:
Maby it needs wrap to structure?

Reply:
Saving that for a big refactoring task when we will move mirror to the `d8` cli